package.json

```diff
@@ -1,36 +1,37 @@
 {
   "name": "business-edit-ui",
-  "version": "4.5.6",
+  "version": "4.6.0",
   "private": true,
   "appName": "Edit UI",
   "sbcName": "SBC Common Components",
   "scripts": {
-    "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint",
-    "lint:nofix": "vue-cli-service lint --no-fix",
-    "test:unit": "vue-cli-service test:unit --testPathPattern --coverage",
-    "test:debug": "node --inspect-brk node_modules/.bin/vue-cli-service test:unit --testPathPattern --no-cache --watch --runInBand"
+    "dev": "vite",
+    "build": "vite build",
+    "serve": "vite preview",
+    "lint": "eslint . --ext .js,.ts,.vue",
+    "test:unit": "vitest run",
+    "test:coverage": "vitest run --coverage"
   },
   "dependencies": {
     "@babel/compat-data": "^7.21.5",
     "@bcrs-shared-components/action-chip": "1.1.5",
     "@bcrs-shared-components/approval-type": "1.0.19",
+    "@bcrs-shared-components/base-address": "2.0.3",
     "@bcrs-shared-components/breadcrumb": "2.1.5",
-    "@bcrs-shared-components/business-lookup": "1.1.7",
+    "@bcrs-shared-components/business-lookup": "1.1.30",
```

Comment:
Updating this one by so much might cause issues -- make sure you test this and/or call it out in the ticket for Riyaz to have a good look at.

Ditto for completing-party below.

The others _should_ be OK since we generally only add to corp types, enums and interfaces (and we'd see code errors for anything amiss there).

Reply:
Sev, I've tested. They look OK.

Business lookup:
![lookup part 1](https://github.com/bcgov/business-edit-ui/assets/122301442/7fae38d9-50a2-44fa-a9e8-71567211ee57)
![lookup part 2](https://github.com/bcgov/business-edit-ui/assets/122301442/f4314c28-ed69-425c-b086-40b4f0593650)

Completing party looks OK too. 

I'll add a note to Riyaz.